selection_sort.py

```diff
@@ -31,7 +31,18 @@ def selection_sort(lista_numeros: list[int]):
     [1, 1, 3]
     """
 
-    # TODO implemente o código fazendo ordenação baseado na ideia do selection sort.
+   for index in range(0, len(lista_numeros)):
+        min_index = index
+
+        for right in range(index + 1, len(lista_numeros)):
+           if lista_numeros[right] < lista_numeros[min_index]:
+               min_index = right
+
+    lista_numeros[index], lista_numeros[ min_index] = lista_numeros[min_index], lista_numeros[index]
+    lista_ordenada_selection = lista_numeros
+    return lista_ordenada_selection
```

Comment:
Tem alguns problemas de indentação aqui:
```suggestion
    for index in range(0, len(lista_numeros)):
        min_index = index

        for right in range(index + 1, len(lista_numeros)):
            if lista_numeros[right] < lista_numeros[min_index]:
                min_index = right

        lista_numeros[index], lista_numeros[ min_index] = lista_numeros[min_index], lista_numeros[index]
    lista_ordenada_selection = lista_numeros
    return lista_ordenada_selection
```

Reply:
Ainda mais importante é a linha `41` que se não estiver no bloco do for da linha `34` não tem como fazer as mudanças na lista.